INSTALL.adoc

```diff
@@ -2,22 +2,33 @@
 
 == Prerequisites
 
-* The GNU C Compiler (gcc) is recommended, as the bytecode interpreter takes
+* A C Compiler is required.
+  The GNU C Compiler (`gcc`) is recommended as the bytecode interpreter takes
   advantage of GCC-specific features to enhance performance. gcc is the standard
-  compiler under Linux, OS X, and many other systems.
+  compiler under Linux and many other systems.
+  However `clang` - used in Mac OS, BSDs and others - also works fine.
+
+* GNU `make`, as well as POSIX-compatible `awk` and `sed` are required.
+
+* A POSIX-compatible `diff` is necessary to run the test suite.
+
+* The GNU BFD library is necessary for the `ocamlobjinfo` tool to be able
+  to process `.cmxs` files. (optional)
 
 * If you do not have write access to `/tmp`, you should set the environment
   variable `TMPDIR` to the name of some other temporary directory.
 
+== Prerequisites (special cases)
+
 * Under HP/UX, the GNU C Compiler (gcc), the GNU Assembler (gas), and GNU Make
   are all *required*.  The vendor-provided compiler, assembler and make tools
   have major problems.
```

Comment:
That was relevant in the late 1990's.  Nowadays I don't know who is still using HP-UX nor how good/bad their tools are.  So, feel free to remove this item.

Reply:
Ok, I removed it then.